docs/content/direct/wds.md

```diff
@@ -2,68 +2,85 @@
 
 - [What is a WDS?](#what-is-a-wds)
 - [Creating a WDS](#creating-a-wds)
+
   - [Using the KubeFlex CLI](#using-the-kubeflex-cli)
-- [KubeFlex Hosting Cluster as WDS](#kubeflex-hosting-cluster-as-wds)
+  - [Accessing the WDS](#accessing-the-wds)
+
+- [Working with a WDS](#working-with-a-wds)
 - [WDS vs. ControlPlane Registration](#wds-vs-controlplane-registration)
 - [Controllers Running in a WDS](#controllers-running-in-a-wds)
-- [Working with a WDS](#working-with-a-wds)
-  - [Accessing the WDS](#accessing-the-wds)
 
-A Workload Description Space (WDS) is a core component of the KubeStellar architecture that serves as the primary interface for users to define and manage workloads for multi-cluster deployment.
+
 
 ## What is a WDS?
 
-A WDS is a space (a Kubernetes-like API server with storage) that:
+A Workload Description Space (WDS) is a space in the [KubeStellar architecture](architecture.md) that serves as the primary interface for users to define and manage workloads for multi-cluster deployment. The WDS consists of a Kubernetes API server with storage that:
 
 - Stores the definitions of workloads in their native Kubernetes format
-- Hosts the control objects (`BindingPolicy` and `Binding`) that define how workloads are distributed
+- Stores the control objects (`BindingPolicy`, `Binding`, `Status Collector`, `CombinedStatus` and `CustomTransform`) that define how workloads are distributed
 - Maintains status information about deployed workloads
 - Acts as the main user interface to the KubeStellar system
 
+
 ## Creating a WDS
 
-A WDS can be created in several ways:
+Currently, we support the use of our Kubestellar core Helm chart as the only way to  create a WDS. This is because the core Helm chart also automatically creates a kubestellar-controller-manager and transport-controller, which are contorllers that your WDS requires to function properly.
 
-### Using the KubeStellar Core Helm Chart
+### Use the KubeStellar Core Helm Chart to create your WDS
 
 The recommended approach is to use the KubeStellar Core Chart:
 
 ```shell
 helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart \
-  --set-json='WDSes=[{"name":"wds1", "type":"k8s"}]'
+  --set-json='WDSes=[{"name":"<wds-name>", "type":"<space-type>"}]'
 ```
 
-You can customize your WDS by specifying:
-- `name`: A unique name for the WDS
-- `type`: 
-  - `k8s` (default): Creates a basic Kubernetes API Server with a subset of kube controllers
-  - `host`: Uses the KubeFlex hosting cluster itself
-- `APIGroups`: A comma-separated list of API Groups to include
-- `ITSName`: The name of the ITS to be used by this WDS (required if multiple ITSes exist)
+Specify a `name` for your WDS, and you can customize it further with optional parameters:  
 
-### Using the KubeFlex CLI
+- `name` (required): A unique name for the WDS (`<wds-name>`)
```

Comment:
You no longer need "name" in this list of parameters, you already described it just above.